transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/BiosampleTransformations.scala

```diff
@@ -38,7 +50,21 @@ object BiosampleTransformations {
       replicationType = "repType",
       treatments = biosampleInput.read[Array[String]]("treatments"),
       wasPerturbed = biosampleInput.read[Boolean]("perturbed"),
-      submittedBy = biosampleInput.read[String]("submitted_by")
+      submittedBy = biosampleInput.read[String]("submitted_by"),
+      productId = if (productIds.size > 1) {
+        logger.warn(
+          s"Biosample '$biosampleId' has multiple product ids: [${productIds.mkString(",")}]."
+        )
+        None
+      } else {
+        productIds.headOption
+      },
+      lotId = if (productIds.size > 1) {
```

Comment:
```suggestion
      lotId = if (lotIds.size > 1) {
```

Reply:
Ah, darn 🤦‍♀️, yes good catch. Thank you!